bgpd/bgp_updgrp_packet.c

```diff
@@ -579,19 +597,29 @@ struct stream *bpacket_reformat_for_peer(struct bpacket *pkt,
 			}
 
 
-			if (nhlen == BGP_ATTR_NHLEN_IPV6_GLOBAL_AND_LL
-			    || nhlen == BGP_ATTR_NHLEN_VPNV6_GLOBAL_AND_LL) {
+			if ((nhlen == BGP_ATTR_NHLEN_IPV6_GLOBAL_AND_LL ||
+				nhlen == BGP_ATTR_NHLEN_VPNV6_GLOBAL_AND_LL) ||
+				(bgp_flag_check(peer->bgp,
```

Comment:
if `BGP_FLAG_IPV6_LINK_LOCAL_ONLY` not set, is this part handled as well?

Reply:
yes at below line no 613